README.md

```diff
@@ -37,30 +37,56 @@ dumps
 └── users
     └── index
         └── 0.json
+        └── 0.request_body
+        └── 0.response_body
```

Comment:
From https://github.com/Pioneer-Valley-Books/rails-response-dumper/pull/82#discussion_r1481922149

> I also think we should drop "_body" from these file names as I don't find it adds any context but it does add general noise to the directory.

```suggestion
        └── 0.request
        └── 0.response
```

Reply:
I have concern with removing the suffix, as the latter file (for example) is not "the response," but only the body of the response, so this feels a little misleading, especially to those skimming/learning the gem. Given that "0.json" provides no information of this kind on its own, the clarity feels even more warranted.

Reply:
Okay, we can go ahead with the verbose suffix.